src/Templating/src/Microsoft.DotNet.Web.ProjectTemplates/content/EmptyWeb-CSharp/Program.cs

```diff
@@ -14,11 +15,14 @@ public class Program
     {
         public static void Main(string[] args)
         {
-            CreateWebHostBuilder(args).Build().Run();
+            CreateHostBuilder(args).Build().Run();
         }
 
-        public static IWebHostBuilder CreateWebHostBuilder(string[] args) =>
-            WebHost.CreateDefaultBuilder(args)
-                .UseStartup<Startup>();
+        public static IHostBuilder CreateHostBuilder(string[] args) =>
```

Comment:
Not for this PR, but I'd like us to revisit the `CreateHostBuilder` pattern completely in 3.0, i.e. question whether we still think it's worth the potential confusion of an untyped, pattern-based alternate entry point.

Reply:
Agreed. I think EF still relies on this though.